extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -188,7 +188,6 @@ ConsumeWindowsEventLog::ConsumeWindowsEventLog(const std::string& name, const ut
 }
 
 void ConsumeWindowsEventLog::notifyStop() {
-  std::lock_guard<std::mutex> lock(on_trigger_mutex_);
   logger_->log_trace("start notifyStop");
   bookmark_.reset();
```

Comment:
could an `onTrigger` be running when this `notifyStop` is called? if it could, the mutex protects the bookmark so we might not be able to remove it